test/DebtToken.t.sol

```diff
@@ -69,9 +69,17 @@ contract DeploymentTest is DebtTokenTest {
 
     //Deployment
     function testSucces_Deployment() public {
+        // Given: all neccesary contracts are deployed on the setup
+
+        // When: debt is DebtToken
+
+        // Then: debt's name should be Arcadia Asset Debt
         assertEq(debt.name(), string("Arcadia Asset Debt"));
+        //And: debt's symbol should be darcASSET
         assertEq(debt.symbol(), string("darcASSET"));
+        //And: debt's decimals should be 18
```

Comment:
these can be added to the Then at line 76 with commas

Reply:
All and can be shorten like that

Reply:
Will change it like this